contractapi/contract_chaincode.go

```diff
@@ -231,7 +231,7 @@ func (cc *ContractChaincode) addContract(contract ContractInterface, excludeFunc
 	}
 
 	if _, ok := cc.contracts[ns]; ok {
-		return fmt.Errorf("Multiple contracts being merged into chaincode with name %s", contract.GetName())
+		return fmt.Errorf("Multiple contracts being merged into chaincode with name %s", ns)
```

Comment:
why have you changed this here? you haven't changed it everywhere in this file..

Reply:
The if above works out what to call the contract if GetName returns blank I realized when adding the new error message that the error was using the wrong value and therefore would be blank. Unlikely to hit it as a problem as go would prevent you having two types with the same name in the same package but better for neatness